src/tool/cppwinrt/cppwinrt.props

```diff
@@ -0,0 +1,62 @@
+<Project DefaultTargets="Build" ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+<!-- Copyright (c) Microsoft. All rights reserved.
+Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+This file sets PlatformToolset/WindowsTargetPlatformVersion to allow this project to compile with multiple versions
+of Visual Studio.
+-->
+
+<PropertyGroup>
+    <PlatformToolset>v140</PlatformToolset>
+    <PlatformToolset Condition="'$(VisualStudioVersion)' == '15.0'">v141</PlatformToolset>
+    <PlatformToolset Condition="'$(VisualStudioVersion)' == '16.0'">v142</PlatformToolset>
+    <CppWinRTVersion Condition="'$(CppWinRTVersion)'==''">2.3.4.5</CppWinRTVersion>
```

Comment:
seems arbitrary - maybe 2.0.0.0 ?

Reply:
It was intentional to test the generation of the rc file.